src/v3/src/components/Select/Select.tsx

```diff
@@ -69,93 +64,50 @@ const Select: UISchemaElementComponent<UISchemaElementComponentWithValidationPro
   };
 
   return (
-    <FormControl
-      disabled={loading}
-      error={hasErrors}
+    <NativeSelect
+      autoCompleteType={autocomplete}
+      errorMessage={errorMessage}
+      errorMessageList={errorMessageList}
+      hint={!required ? optionalLabel : undefined}
```

Comment:
Why not use `isOptional` prop instead of `hint`?

Reply:
Not sure how this happened. I think one of the other components I migrated didn't support `isOptional` so I was forced to do this and then I just assumed that `NativeSelect` doesn't support it as well. I'll change that here and in the `PhoneAuthenticator` migration, nice catch